tests/e2e/lib.sh

```diff
@@ -43,42 +43,71 @@ deploy_stackrox() {
 }
 
 # shellcheck disable=SC2120
-deploy_stackrox_with_custom_sensor() {
-    if [[ "$#" -ne 1 ]]; then
-        die "expected sensor chart version as parameter in deploy_stackrox_with_custom_sensor"
+deploy_stackrox_with_custom_central_and_sensor_versions() {
+    if [[ "$#" -ne 2 ]]; then
+        die "expected central chart version and sensor chart version as parameters in deploy_stackrox_with_custom_central_and_sensor_versions: deploy_stackrox_with_custom_central_and_sensor_versions <central chart version> <sensor chart version>"
     fi
-    target_version="$1"
-    setup_podsecuritypolicies_config
-
-    deploy_central
+    ci_export CENTRAL_CHART_VERSION_OVERRIDE "$1"
+    ci_export SENSOR_CHART_VERSION_OVERRIDE "$2"
```

Comment:
(*) With chart overrides now done via directories, you don't need to push version overrides to further scripts. I suggest removing these.
```suggestion
```

Moreover, these variables are already present in the environment because the calling function checks their presence.
https://github.com/stackrox/stackrox/pull/5795/files#diff-940ad3f62ebd4b3bf691cac553e159b376ebd82a4a436e020e49b22503eef578R23-R25

Reply:
When I had previously removed `SENSOR_CHART_VERSION_OVERRIDE` the style check had complained that it wasn't defined, but maybe that was because there was also a `sensor_chart_version_override` present. I'll give removing these a try and see if quickstyle passes.